creativecloud/scripts/utils.js

```diff
@@ -58,13 +58,23 @@ function getDecorateAreaFn() {
     if (lcpImg) lcpImgSet = true;
   };
 
+  function replaceDotMedia(area = document) {
+    const resetAttributeBase = (tag, attr) => {
+      area.querySelectorAll(`${tag}[${attr}^="./media_"]`).forEach((el) => {
+        el[attr] = `${new URL(`${getConfig().contentRoot}${el.getAttribute(attr).substring(1)}`, window.location).href}`;
```

Comment:
I wonder if you want to use the contentRoot or just prepend `./cc-shared/` to the attribute so that it's something like `./media/cc-shared`

Reply:
We agreed on using content root first in the path in #milo-dev and same is being used in homepage too